src/dapps/Unstoppable/containers/ManageUnstoppable/helpers.ts

```diff
@@ -0,0 +1,72 @@
+export default {
+  BTC: 'crypto.BTC.address',
```

Comment:
this is duplicated in src/dapps/Unstoppable/containers/ManageCryptoContainer/helpers.js